sync/class.jetpack-sync-module-plugins.php

```diff
@@ -24,6 +26,25 @@ public function init_before_send() {
 		//Note that we don't simply 'expand_plugin_data' on the 'delete_plugin' action here because the plugin file is deleted when that action finishes
 	}
 
+	public function install_plugin( $upgrader ) {
+		if ( 'Plugin_Upgrader' === get_class( $upgrader ) ) {
```

Comment:
Do we get any more info then what the upgrader class is to determine what action was take?